Source/monster.cpp

```diff
@@ -223,11 +221,11 @@ void InitMonster(Monster &monster, Direction rd, int mtype, Point position)
 		monster.mLevel += 15;
 		monster.mExp = 2 * (monster.mExp + 1000);
 		monster.mHit += NIGHTMARE_TO_HIT_BONUS;
-		monster.mMinDamage = 2 * (monster.mMinDamage + 2);
-		monster.mMaxDamage = 2 * (monster.mMaxDamage + 2);
+		monster.mDamage += { 2 };
+		monster.mDamage *= 2;

```

Comment:
I'm assuming this was split onto two lines due to only having operator overloads for `Damage, int` and not `int, Damage`, this should work:
```suggestion
		monster.mDamage = (monster.mDamage + 2) * 2;
```

`operator+` and the `Damage(int)` constructor should promote the int in addition... if not adding an explicit overload for `+ int` would be worth it given it has a lot of uses

Reply:
I didn't want to repeat the value like that so the use of the 2 operations in sequence was on purpose.